lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -2434,5 +2434,124 @@ def test_cell_method_correct_order(self):
         self.assertTrue(cube1 == cube2)
 
 
+class Test__summary_dim_name(tests.IrisTest):
+    def test_standard_named_dim(self):
+        cube = Cube(np.zeros((5, 3)))
+        cube.add_dim_coord(
+            DimCoord(np.arange(5), standard_name="longitude", units="degrees"),
+            0,
+        )
+        result = cube._summary_dim_name(0)
+        self.assertEqual(result, "longitude")
+
+    def test_long_named_dim(self):
+        cube = Cube(np.zeros((5, 3)))
+        cube.add_dim_coord(
+            DimCoord(np.arange(5), long_name="alons", units="degrees"), 0
+        )
+        result = cube._summary_dim_name(0)
+        self.assertEqual(result, "alons")
+
+    def test_unmapped_dim(self):
+        cube = Cube(np.zeros((5, 3)))
+        cube.add_dim_coord(DimCoord(np.arange(5), long_name="x", units=1), 0)
+        result = cube._summary_dim_name(1)
+        self.assertEqual(result, "-- ")
+
+    def test_aux_mapped_dim(self):
+        # It should *not* treat an aux-coord as the name of a dim.
+        cube = Cube(np.zeros((5, 3)))
+        cube.add_aux_coord(
+            DimCoord(np.arange(5), standard_name="longitude", units="degrees"),
+            0,
+        )
+        result = cube._summary_dim_name(0)
+        self.assertEqual(result, "-- ")
+
+
+class Test__summary_vector_sections_info(tests.IrisTest):
```

Comment:
Need something to confirm the [specialised handling of `scalar_coords`](https://github.com/SciTools/iris/pull/3914/files#diff-38eaa350bf479c486e9f6659b8077be2d52448f17739872e50285b98b5423415R2321-R2339).

Reply:
Ok!